rocketpy/plots/rocket_plots.py

```diff
@@ -104,6 +105,181 @@ def thrust_to_weight(self):
 
         return None
 
+    def draw(self, vis_args=None):
+        """Draws the rocket in a matplotlib figure.
+
+        Parameters
+        ----------
+        vis_args : dict, optional
+            Determines the visual aspects when drawing the rocket. If None,
+            default values are used. Default values are:
+            {
+                "background": "#EEEEEE",
+                "tail": "black",
+                "nose": "black",
+                "body": "dimgrey",
+                "fins": "black",
+                "motor": "black",
+                "buttons": "black",
+                "line_width": 2.0,
+            }
+            A full list of color names can be found at:
+            https://matplotlib.org/stable/gallery/color/named_colors
+        """
+        if vis_args is None:
```

Comment:
I'd like an opinion on this trade-off:

Is it better to let this standard dictionary as the default value of the method? Or just let it as it is here, checking if vis_args was passed?

I believe the Flight.export_data() method does a similar approach to this.

Reply:
Adding it to the default argument value may simplify documentation. But since you are already describing it in the docstring, I believe the current implementation is fine.